Ownership/AC - West/8.txt

```diff
@@ -3,8 +3,14 @@ OWNER:LW8:LWL:LWB:LW:LWN:L
 ALTOWNER:Observing LON_W_CTR:LGS:LW8:LWL:LWB:O:LMEN:LW:LWN:L
 ALTOWNER:Observing London FIR:LGS:O:LMEN:LW8:LWL:LWB:LW:LWN:L
 
-SECTOR:London S8 (DB-335):0:33500
+SECTOR:London S8 01 (DB-355):0:35500
 OWNER:LW8:LWL:LWB:LW:LWN:L
 ALTOWNER:Observing LON_W_CTR:LGS:LW8:LWL:LWB:O:LMEN:LW:LWN:L
 ALTOWNER:Observing London FIR:LGS:O:LMEN:LW8:LWL:LWB:LW:LWN:L
-BORDER:LW04:LW09:LW10:LW12:LW14:BAN04:BAN05
+BORDER:LWN01:LWN02:LW10:LW09:LW04:BAN03:LW14:LW13
+
+SECTOR:London S8 BADSI-LIPGO (245-355):24500:35500
```

Comment:
Just wondering if we still need this section since the BADSI-LIPGO section has been removed from S35? Delegation ends above FL245 anyway, is this an ES workaround or just here because it is?